Context on concurrent Cypherl import (#552)
memgraph/documentation: pages/data-migration/cypherl.mdx

This is how the graph should look like in Memgraph after the import:

![](/pages/data-migration/cypherl/cypherl_multiple_type_nodes_and_relationships.png)

## Concurrent import

> If you are using a [client libraries](../client-libraries.mdx) to import data from your Cypherl file, 

Reviewer: ```suggestion
If you are using a [client library](../client-libraries.mdx) to import data from your CYPHERL file, 
```

you can leverage concurrent import to speed up the import process. 

> You would split a file into smaller chunks of nodes and relationships, which can be imported concurrently. Here is the basic flow: 

Reviewer: ```suggestion
Split a file into smaller chunks of nodes and relationships, which can be imported concurrently. Here is the basic flow: 
```

> 1. Open and read the file with the Cypherl queries.
> 2. Split the .cyherl file into a list of nodes and relationships.
> 3. Split the list of nodes and relationships into smaller chunks.
> 4. Import the node's chunks concurrently.
> 5. Import the relationships concurrently.

Reviewer: ```suggestion
1. Open and read the file with the Cypher queries.
2. Split the CYPHERL file into a list of nodes and relationships.
3. Split the list of nodes and relationships into smaller chunks.
4. Import node chunks concurrently.
5. Import relationship chunks concurrently.
```

> For details on how to achieve this, please refer to the documentation of the [client library](../client-libraries.mdx) you are using and best practice for import 
> the section about Cypherl import.

Reviewer: ```suggestion
For details on how to achieve this, please refer to the documentation of the [client library](../client-libraries.mdx) you are using.
```

Reviewer: We'll add links later
